src/macro-expansion.md

```diff
@@ -515,20 +516,20 @@ We use these items in macro parser:
   ask the MBE parser to parse. We will consume the raw stream of tokens and
   output a binding of metavariables to corresponding token trees. The parsing
   session can be used to report parser errors.
-- `ms` a _matcher_. This is a sequence of token trees that we want to match
-  the token stream against.
+- `matcher` is a sequence of `MatcherLoc`s that we want to match
+  the token stream against. They're converted from token trees before matching.
```

Comment:
Not sure the updated doc is correct, cc @petrochenkov could you review if you have some time?

Reply:
I'm not sure about this part either.

Reply:
Looks correct.